app/views/public/api/v1/portals/_header.html.erb

```diff
@@ -8,34 +7,161 @@
-            </span>
-          </a>
+        <div class="ml-8 border-l-1 border-slate-50 dark:border-slate-800 hidden md:block cursor-pointer px-1 py-2">
+          <div class="flex-grow flex-shrink-0">
+            <a target="_blank" rel="noopener noreferrer nofollow" href="<%= @portal.homepage_link %>" class="flex flex-row items-center gap-1 text-sm font-medium text-slate-800 dark:text-slate-100 hover:text-woot-500 dark:hover:text-woot-500 stroke-slate-700 dark:stroke-slate-200 hover:stroke-woot-500 dark:hover:stroke-woot-500">
+              <svg class="w-4 h-4" width="15" height="16" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M6.5 3H3.5C2.94772 3 2.5 3.44772 2.5 4V12C2.5 12.5523 2.94772 13 3.5 13H11.5C12.0523 13 12.5 12.5523 12.5 12V9" stroke-width="1.2" stroke-linecap="round"/>
+                <path d="M9.5 3H12.3C12.4105 3 12.5 3.08954 12.5 3.2V6" stroke-linecap="round"/>
+                <path d="M12 3.5L6 9.5" stroke-width="1.2" stroke-linecap="round"/>
+              </svg>
+              <%= I18n.t('public_portal.header.go_to_homepage') %>
+            </a>
+          </div>
         </div>
-      </div>
       <% end %>
-    </div>
-    <% if @portal.config["allowed_locales"].length > 1 %>
-    <div class="flex items-center">
-      <div class="inline-flex relative w-24">
-        <select
-          data-portal-slug="<%= @portal.slug %>"
-          class="appearance-none w-full bg-white dark:bg-slate-900 dark:text-slate-100 px-3 py-2 pr-8 rounded leading-tight focus:outline-none focus:shadow-outline locale-switcher hover:bg-slate-75 dark:hover:bg-slate-800 cursor-pointer"
-        >
-          <% @portal.config["allowed_locales"].each do |locale| %>
-            <option <%= locale == params[:locale] ? 'selected': '' %> value="<%= locale %>"><%= locale %></option>
-          <% end %>
-        </select>
-        <div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-slate-700 dark:text-slate-200">
-          <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"/></svg>
+
+      <%# Appearance toggle section %>
+      <div class="flex-grow flex-shrink-0 relative cursor-pointer px-1 py-2">
+        <button class="toggle-appearance flex flex-row items-center stroke-slate-700 dark:stroke-slate-200 hover:stroke-woot-500 dark:hover:stroke-woot-500 text-slate-800 dark:text-slate-100 hover:text-woot-500 dark:hover:text-woot-500 gap-1" type="button" onclick="toggleAppearanceDropdown()">
+          <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we figure out something else here? The usage of SVG like this is a bit off-putting. Let me know.

Reply:
@pranavrajs Done https://github.com/chatwoot/chatwoot/pull/8089/commits/2eff4e9b470579eb83e08d8466dccb7422179df3